src/docfx/build/xref/XrefResolver.cs

```diff
@@ -215,12 +216,28 @@ orderby monikerGroup
         return model;
     }
 
-    private IReadOnlyDictionary<string, InternalXrefSpec[]> BuildInternalXrefMap()
+    public Dictionary<string, ExternalXrefSpec> ResolveXrefMapByFile(
```

Comment:
It seems better to return `InternalXrefSpec` as the rest of the interface.

Reply:
@herohua  
this method is designed to output the result which is similar to _XrefResolver.ToXrefMapModel()_ method 
https://github.com/freewheel70/docfx/blob/ad3176f1383257b054b5094fb7379e2d734f253a/src/docfx/build/xref/XrefResolver.cs#L154 , therefore ExternalXrefSpec is more suitable. 

FYI, the latest signature of this  method is 
```
public List<ExternalXrefSpec> ResolveXrefMapByFile(FilePath file)
```